warehouse/warehouse.go

```diff
@@ -578,25 +589,6 @@ func (wh *HandleT) processJobs(warehouse warehouseutils.WarehouseT) (err error)
 		delete(inRecoveryMap, warehouse.Destination.ID)
 	}
 
-	// Step 2: Handle any Pending uploads
-	// An upload job is pending if it is neither exported nor aborted
-
-	pendingUploads, err := wh.getPendingUploads(warehouse)
```

Comment:
This functions definition can be removed since its not used anywhere else

Reply:
removed